package.json

```diff
@@ -49,5 +50,9 @@
     "sinon-chai": "~3.2",
     "ts-node": "~7.0",
     "typescript": "~3.1"
+  },
+  "engines": {
+    "node": ">11.0.0",
```

Comment:
Since this is a package to be included in another project, IMO we shouldn't add `engines` as it limits what people how people can use the software - this would be ok for me in a build system, not a package.

That said I understand the point to try to force developers of this package to use these versions to commit their code (and I agree we should), but this will also limit end users. Maybe there is another way to make this happen without using `engines`?

If we really have to include `engines`, I would suggest the following, as these are the LTS versions:
```
"node": ">=8.12.0 <9.0.0 || >=10.11.0",
```